content/blog/2022-03-whats-new-in-axum-0-5.md

```diff
@@ -0,0 +1,298 @@
+---
+date: "2022-04-31"
+title: "What's new in axum 0.5"
+description: "March 31, 2022"
+---
+
+Today, we're happy to announce [axum] version 0.5. axum is an ergonomic and
+modular web framework built with tokio, tower, and hyper.
```

Comment:
nit: personally, I would either capitalize or use code formatting for library names:
```suggestion
Today, we're happy to announce [`axum`] version 0.5. `axum` is an ergonomic and
modular web framework built with `tokio`, `tower`, and `hyper`.
```